datetime/example_test.go

```diff
@@ -80,6 +80,21 @@ func Example() {
 	fmt.Printf("Data: %v\n", respDt.ToTime())
 }
 
+// ExampleNewDatetime_localUnsupported demonstrates that "Local" location is
+// unsupported.
+func ExampleNewDatetime_localUnsupported() {
+	tm := time.Now().Local()
+	fmt.Println("Location:", tm.Location().String())
+	if _, err := NewDatetime(tm); err != nil {
+		fmt.Println("Could not create a Datetime with Local location.")
+	} else {
+		fmt.Println("A Datetime with Local location created.")
+	}
+	// Output:
+	// Location: Local
+	// Could not create a Datetime with Local location.
```

Comment:
Is there a way to create `Datetime` in Go with `Europe/Moscow` if I'm my local is `Europe/Moscow`?

Reply:
Yeah, time.LoadLocation("Europe/Moscow").Location().String() == "Europe/Moscow".